pages/dkp/konvoy/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -162,11 +162,23 @@ Before running the following command, ensure that your `dkp` configuration **ref
     dkp upgrade kommander -v 4
     ```
 
+    If you find any `HelmReleases` in a "broken" release state such as "exhausted" or "another rollback/release in progress", you can trigger a reconciliation of the `HelmRelease` using the following commands:
+
+    ```bash
+    kubectl -n kommander patch helmrelease <HELMRELEASE_NAME> --type='json' -p='[{"op": "replace", "path": "/spec/suspend", "value": true}]'
+    kubectl -n kommander patch helmrelease <HELMRELEASE_NAME> --type='json' -p='[{"op": "replace", "path": "/spec/suspend", "value": false}]'
+    ```
+
 1.  For Enterprise customers (multi-cluster environment): Upgrade your additional [Workspaces][upgrade_workspaces] on a per-Workspace basis to upgrade the Platform Applications on other clusters than the Management Cluster.
     For Essential customers (single-cluster environment): Proceed with the [Konvoy Upgrade][konvoy_upgrade].
 
 You can always go back to the [DKP Upgrade overview][dkp_upgrade], to review the next steps depending on your environment and license type.
 
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0][https://www.gnu.org/licenses/agpl-3.0.en.html]. The complete source code for the versions of MinIO packaged with DKP 2.2.0 are available at these URLs:
```

Comment:
@gvasquezvargas this might have something to do with recent minio attributions change but i am not sure which "docker image" is being referred here. i just copied over from https://github.com/mesosphere/dcos-docs-site/blob/755e9f849a17b993b99d40ff7f3b4d8e9908b738/pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md?plain=1#L179 to keep things consistent.

Reply:
Good catch! yes, we better have this here as well, just to be sure.